src/components/Typography/TypographyConstants.ts

```diff
@@ -0,0 +1,12 @@
+export enum TypographyColor {
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+  ON_PRIMARY = "onPrimary",
+  ON_INVERTED = "onInverted"
+}
+
+export enum TypographyAlign {
```

Comment:
I thought we decided to remove it

Reply:
we not talked about it as far as I remember, why to remove it?

Reply:
in most cases, layout is related to containers and not the element itself.
for example - a table cell will control the alignment of its content, and not the textual content itself
in other cases - you would probably use flex/position to align content that also contains text